.travis.yml

```diff
@@ -8,8 +8,7 @@ jdk:
   - oraclejdk7
 
 env:
-  - PDB_TEST_LANG=clojure PDB_TEST_DB=hsqldb
-  - PDB_TEST_LANG=clojure PDB_TEST_DB=postgres
```

Comment:
Sorry to be late to the game here, but fyi you'll have to remove this from the jenkins config as well `ext/jenkins/lein-test.sh`